### PR TITLE
UID update tip added to dockerfile README.md

### PR DESCRIPTION
**Pull request by ashendes**
## Related Issues
https://github.com/wso2/docker-is/issues/286
https://github.com/wso2/product-is/issues/12181

### CODE REVIEW

**Review by pulasthi7 — APPROVED** [review]

File: dockerfiles/alpine/is/README.md [hunk 1]
```diff
@@ -19,6 +19,10 @@ git clone https://github.com/wso2/docker-is.git
 
 ##### 2. Build the Docker image.
 
+>Tip - If you require the container to run with a different UID and GID, change the value of `USER_ID` and
+> `USER_GROUP_ID` to the required UID and GID respectively in the arguments section of the dockerfile before building
+> the image. Please note that setting lower values for the UID and GID is not recommended.
```
Comment (gomathyk) [inline dockerfiles/alpine/is/README.md:24]:
```suggestion
> the image. Note that setting lower values for the UID and GID is not recommended.
```

Comment (gomathyk) [reply dockerfiles/alpine/is/README.md:24]: Please get rid of "Please" everywhere.

**Review by gomathyk — APPROVED** [review]

File: dockerfiles/alpine/is/README.md (hunk 1 above)
Comment (ashendes) [reply dockerfiles/alpine/is/README.md:24]: Will do 😆

File: dockerfiles/alpine/is/README.md [hunk 2]
```diff
@@ -25,6 +25,10 @@ git clone https://github.com/wso2/docker-is.git
 
 > By default, the Docker image will prepackage the General Availability (GA) release version of the relevant WSO2 product.
 
+> Tip - If you require the container to run with a different UID and GID, pass the preferred values of the UID and GID
+> as values for build arguments `USER_ID` and `USER_GROUP_ID` when building the image in the manner shown below. Note
```
Comment (gomathyk) [inline dockerfiles/alpine/is/README.md:29]:
```suggestion
> as values for build arguments `USER_ID` and `USER_GROUP_ID` when building the image, as shown below. Note
```

**Pull request merged**